CONTRIBUTING.rst

```diff
@@ -41,5 +41,19 @@ As shown in the template_ module, the minimum requirements for a new feature are
 
 PRs will not be accepted if tests fail.
 
+Important Guidelines
+--------------------
+
+Astroquery is based on the requests module.  All contributions must be based on
+the `requests` module and should not use `urllib` or any of the base python url
+modules unless there is a demonstrated necessity.
+
+The `requests` module also generally should not be directly used, since the
```

Comment:
not sure that sphinx will be happy with these, but we can always fix later if it's an issue.

Reply:
right, should be `requests <url>`_